src/shared/project-output/ManagerProjectOutput.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import graphql from 'babel-plugin-relay/macro';
+import { Box, Grid, Typography } from '@material-ui/core';
+import { Evaluation } from 'src/global-types';
+import { EvaluationOutput } from 'src/shared/evaluation-output';
+import { i18n } from '@lingui/core';
+import { useAuthGuardUser } from 'src/core/auth';
+import { useFragment } from 'react-relay/hooks';
+import { useRoundQuestions } from 'src/core/round-questions';
+
+import { ManagerProjectOutput_review$key } from './__generated__/ManagerProjectOutput_review.graphql';
+import { MultilineOutput } from '../multiline-output';
+import { QuestionOutput } from './QuestionOutput';
+import { ReviewItemInfo } from '../review-item-info';
+import { getQuestionsAnswersPair } from '../utils/questionsAnswersPair';
+import { getUserLabel } from '../utils/getUserLabel';
+
+const fragment = graphql`
+  fragment ManagerProjectOutput_review on ManagerProjectCommentNode {
+    rating
+    answers {
+      questionId
+      value
+    }
+  }
+`;
+
+interface OwnProps {
+  review: ManagerProjectOutput_review$key;
+  self: boolean;
+}
+
+type Props = React.PropsWithChildren<OwnProps>;
+
+export function ManagerProjectOutput(props: Props) {
+  const review = useFragment(fragment, props.review);
+  const { managerReviewProjectQuestions } = useRoundQuestions();
+  const me = useAuthGuardUser();
+  const questionsAnswers = getQuestionsAnswersPair(managerReviewProjectQuestions, review.answers);
+
+  if (!review) {
+    return null;
+  }
+
+  return (
+    <Box bgcolor="success.light" borderRadius={3} p={2}>
+      <ReviewItemInfo name={getUserLabel(me)} src={me.avatarUrl ?? undefined} type="manager">
```

Comment:
in ManagerReviewDominantCharacteristicsExpansionPanel.tsx u use name={getUserLabel(me) 
?? undefined} . i think u can delete this there
maybe u can delete this for me.avatarUrl ?? undefined too

Reply:
I want to use this component in the result phase, so I've designed it more generically.